src/status_im/hardwallet/core.cljs

```diff
@@ -1505,15 +1505,16 @@
         instance-uid (get-in db [:hardwallet :application-info :instance-uid])
```

Comment:
could you add a docstring to on-pair-success that explains what a pairing is and why we need to persist it here?

Reply:
added